src/components/CentralContainer/Sidebar/SidebarTabs/TitleButton.css

```diff
@@ -0,0 +1,5 @@
+.sidebar-tab-title-button {
+    z-index: 0 !important;
```

Comment:
Let's add `--ylv-panel-title-button-z-index: 0;` in `src/index.css` and use it here.